Source/JavaScriptCore/jit/ExecutableAllocator.h

```diff
@@ -114,6 +114,13 @@ ALWAYS_INLINE bool isJITPC(void* pc)
 
 JS_EXPORT_PRIVATE void dumpJITMemory(const void*, const void*, size_t);
 
+// We use this to prevent compile errors on some platforms that are unhappy
+// about the signature of the system's memcpy.
+static ALWAYS_INLINE void* memcpyWrapper(void* dst, const void* src, size_t bytes)
+{
+    return memcpy(dst, src, bytes);
+}
+
```

Comment:
You'll need to put this outside of the `#if ENABLE(JIT)` conditional.  This is the source of the watch build failure.